src/System.Memory/tests/ReadOnlySpan/CopyTo.cs

```diff
@@ -121,5 +122,68 @@ public static void Overlapping2()
             int[] expected = { 90, 92, 93, 94, 95, 96, 97, 97 };
             Assert.Equal<int>(expected, a);
         }
+
+        // This test case tests the Span.CopyTo method for large buffers of size 4GB or more. In the fast path,
+        // the CopyTo method performs copy in chunks of size 4GB (uint.MaxValue) with final iteration copying
+        // the residual chunk of size (bufferSize % 4GB). The inputs sizes to this method, 4GB and 4GB+256B,
+        // test the two size selection paths in CoptyTo method - memory size that is multiple of 4GB or,
+        // a multiple of 4GB + some more size.
+        [Theory]
+        [OuterLoop]
+        [PlatformSpecific(TestPlatforms.Windows | TestPlatforms.OSX)]
+        [InlineData(4L * 1024L * 1024L * 1024L)]
+        [InlineData((4L * 1024L * 1024L * 1024L) + 256)]
+        public static void CopyToLargeSizeTest(long bufferSize)
```

Comment:
Does this test pass on Desktop?

Could you please verify that by doing:
`build.cmd -framework:netfx`
`msbuild <path-to-test-csproj> /t:rebuildandtest /p:targetgroup=netfx`